### PR TITLE
Created new acronyms and abbreviations page

### PR DESCRIPTION
**Pull request by saikatgithub**
Created a new acronyms and abbreviations page and added link at the main readme page

### CODE REVIEW

File: docs/acronyms.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+| FQDN                 | Fully Qualified Domain Name                                      |
+| FS                   | File Server                                                      |
+| GPU                  | Graphics Processing Unit                                         |
+| ML                   | Machine Learning                                                 |
+| MNIST                | Modified National Institute of Standards and Technology Database |
+| MSC                  | My HPE Software Center                                           |
+| PYT                  | PyTorch                                                          |
+| SL                   | Swarm Learning                                                   |
+| SN                   | Swarm Network                                                    |
+| SPIFFE               | Secure Production Identity Framework for Everyone	              |
```
Comment (sathya-SL) [inline docs/acronyms.md:18]: plz align

File: docs/acronyms.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## Acronyms and Abbreviations
+
+| Acronym/Abbreviation | Explanation                                                      |
+| -------------------- | ---------------------------------------------------------------- |
+| API                  | Application Programming Interface                                |
+| APLS                 | Autopass License Server                                          |
+| CIFAR                | Canadian Institute For Advanced Research                         |
+| DTR                  | Docker Trusted Registry                                          |
+| FQDN                 | Fully Qualified Domain Name                                      |
```
Comment (sathya-SL) [inline docs/acronyms.md:9]: FQDN , GPU , ML , API are well-known plz exclude.

File: docs/acronyms.md (hunk 1 above)
Comment (saikatgithub) [reply docs/acronyms.md:18]: Not sure why it is appearing like this here. Actually it looks fine in edit version and actual table.

So ignoring the command.

File: docs/acronyms.md (hunk 2 above)
Comment (saikatgithub) [reply docs/acronyms.md:9]: Removed

File: docs/acronyms.md (hunk 1 above)
Comment (saikatgithub) [reply docs/acronyms.md]: By the way - Looks like it is fixed with new commits. Please verify.

**Review by sharvanideshmukh — APPROVED** [review]

**Review by suresh-ls — APPROVED** [review]

**Pull request merged**